test/test_helper.exs

```diff
@@ -1 +1,3 @@
+Application.put_env(:beacon, :data_source, DummyApp.BeaconDataSource)
```

Comment:
Lets move this to `config/test.exs` for now.

Reply:
moved to `config/config.ex` where other configs are set atm. We don't need `config/test.exs` I guess 🤔

Reply:
Let's add the standard `import_config "#{Mix.env()}.exs"` to `config.exs` and add the three `.exs` files, and put this config in `test.exs` so we don't cause runtime issues when we are in dev or prod mode, since that module is not compiled in those environments.